config-controller/api/v1alpha1/policy_types.go

```diff
@@ -40,7 +40,6 @@ type SecurityPolicySpec struct {
 	// +kubebuilder:validation:Pattern=`^[^\n\r\$]{5,128}$`
 	// PolicyName is the name of the policy as it appears in the API and UI.  Note that changing this value will rename the policy as stored in the database.  This field must be unique.
 	PolicyName string `json:"policyName"`
-	// +kubebuilder:validation:Required
```

Comment:
Isn't this requiring `policyName` ?

It was `description` that gave me the error.

Reply:
Line 39 is for policyName. This one is for description. Comments before the field apply to the field.

Reply:
Ugh, preceding comments as decorators?

I am asleep and this is a nightmare, isn't it?

Reply:
The nightmare will be over once you hit the approve button.